docs/development/cadence-downstream-dependencies.md

```diff
@@ -1,38 +0,0 @@
-# Updating the Cadence Downstream Dependencies / Releasing the CLI
```

Comment:
Can we keep this file? It's still helpful as a reference.

Reply:
Yes, I'll PR this back in.